synapseAnnotations/data/experimentalData.json

```diff
@@ -941,6 +941,11 @@
         "description": "",
         "source": ""
       },
+      {
+        "value": "AffymetrixHuman1.0STS",
```

Comment:
I wasn't able to find an Affymetrix Human 1.0 STS mentioned anywhere with a google search. Are you sure this is not Affymetrix Human ST? 

GEO is a great source for platform sources: https://www.ncbi.nlm.nih.gov/geo/browse/?view=platforms